text/chapter1.md

```diff
@@ -134,6 +134,7 @@ This book aims to provide an introduction to the PureScript language for beginne
 If you get stuck at any point, there are a number of resources available online for learning PureScript:
 
 - The `#purescript` and `#purescript-beginners` FP Slack channels are a great place to chat about issues you may be having. [Use this link](https://fpchat-invite.herokuapp.com/) to gain access the Slack chatrooms.
+- The [Purescript Discourse Forum](https://discourse.purescript.org/) is another good place to search for solutions to common problems. If you ask your questions here, they will be available to help future readers (whereas on Slack they will be deleted).
```

Comment:
This makes it sound like Slack message history is being lost as a result of an active, deliberate action, which is not really the case. I would put it as "whereas on Slack, message history is only kept for approximately 2 weeks" or something.

Reply:
Applied this update